mmv1/third_party/terraform/tests/resource_container_node_pool_test.go.erb

```diff
@@ -2723,3 +2765,60 @@ resource "google_container_node_pool" "np" {
 }
 
 
+func testAccContainerNodePool_concurrentCreate(cluster, np1, np2 string) string {
+	return fmt.Sprintf(`
+resource "google_container_cluster" "cluster" {
+  name               = "%s"
+  location           = "us-central1-a"
+  initial_node_count = 3
+}
+
+resource "google_container_node_pool" "np1" {
+  name               = "%s"
+  location           = "us-central1-a"
+  cluster            = google_container_cluster.cluster.name
+  initial_node_count = 2
+}
+
+resource "google_container_node_pool" "np2" {
+	name               = "%s"
+	location           = "us-central1-a"
+	cluster            = google_container_cluster.cluster.name
+	initial_node_count = 2
+  }
+`, cluster, np1, np2)
+}
+
+func testAccContainerNodePool_concurrentUpdate(cluster, np1, np2 string) string {
+	return fmt.Sprintf(`
+resource "google_container_cluster" "cluster" {
+  name               = "%s"
+  location           = "us-central1-a"
+  initial_node_count = 3
+}
+
+resource "google_container_node_pool" "np1" {
+  name               = "%s"
+  location           = "us-central1-a"
+  cluster            = google_container_cluster.cluster.name
+  initial_node_count = 2
+  upgrade_settings {
+    max_surge = 1
```

Comment:
```suggestion
    max_surge = 2
```

This defaulted to `1`, triggering https://github.com/hashicorp/terraform-provider-google/issues/13157. If we change the value, we'll subvert the issue.

Reply:
Nice catch! Done.